name = "Could Use Try";
description = "Some commands may raise exception. It is recommended to use the try/catch block to intercept those exception, and process them.

* / : DivisionByZeroError
* % : DivisionByZeroError
* intdiv() : DivisionByZeroError
* << : ArithmeticError
* >> : ArithmeticError
* Phar::mungserver : PharException
* Phar::webphar : PharException

See also `Predefined Exceptions <http://php.net/manual/en/reserved.exceptions.php>`_,
         `PharException <http://php.net/manual/en/class.pharexception.php>`_.
";
clearphp = "";
exakatSince = "1.5.0";